### PR TITLE
feat(style): pull in theme service and pre-loader

### PR DESCRIPTION
**Pull request by christianmemije**
### What's included?

- Pull in theme and pre-loader from ui-platform
- Update Angular on the way

#### Test Steps

- [ ] `npm ci`
- [ ] `npm run serve`
- [ ] Login and test this view

#### Definition of Done

- [ ] Adheres to [Covalent](https://teradata.github.io/covalent/), [Angular-Material](https://material.angular.io/) & [Material Design](https://material.io/guidelines/) patterns
- [ ] All user-facing strings are properly stored in `/src/assets/i18n/en-US.json`
- [ ] Changes are a11y compliant
- [ ] Responsiveness is handled
- [ ] `npm run lint` passes (runs tslint, stylelint, prettier)
- [ ] `npm run test` (unit tests)
- [ ] `npm run e2e` (e2e tests)
- [ ] `npm run build:prod` (AOT, product build test)

### CODE REVIEW

File: src/vantage-ui-template/index.html [hunk 1]
```diff
@@ -21,244 +21,10 @@
...
-        </div>
-      </div>
-    </td-app>
-    <script>
-      var themeLSKey = 'vantage.theme';
-      var defaultTheme = 'light-theme';
-      localStorage.setItem(themeLSKey, localStorage.getItem(themeLSKey) || defaultTheme);
-      document.querySelector('html').classList.add(localStorage.getItem(themeLSKey));
-    </script>
+    <div id="td-pre-loader"></div>
```
Comment (emoralesb05) [inline src/vantage-ui-template/index.html]: Werent we gonna try and make this dynamic? Or where did that end?

**Comment by emoralesb05** [comment]
Also, we really need to change how the theme works to more observable based returning the object we need with the logo, monaco theme and what not. (separate thing tho)

File: src/vantage-ui-template/index.html (hunk 1 above)
Comment (christianmemije) [reply src/vantage-ui-template/index.html]: Will be implemented in https://github.com/Teradata/vantage-ui-platform/pull/51

**Comment by christianmemije** [comment]
> Also, we really need to change how the theme works to more observable based returning the object we need with the logo, monaco theme and what not. (separate thing tho)

Opened [an issue](https://github.com/Teradata/vantage-ui-template/pull/122#issuecomment-582678754) for this in UI platform since it needs more planning.

**Comment by christianmemije** [comment]
Waiting on https://github.com/Teradata/vantage-ui-platform/pull/54

**Comment by christianmemije** [comment]
Addressed both pre-loader and theme comments in ui-platform PRs: teradata/vantage-ui-platform#51 teradata/vantage-ui-platform#54, so ready for re-review.

File: src/vantage-ui-template/index.html (hunk 1 above)
Comment (christianmemije) [reply src/vantage-ui-template/index.html]: Done

**Pull request merged**